### DISCUSSION TITLE
Handlebars parse template that contain errors

### DISCUSSION THREAD

**Discussion by harishajdarevic** [handlebars-lang/handlebars.js#1784] [Q&A]
For example if I've template like this:

"{{ Hello }} {{ World"

The parser would return an error. 

Is there a way to parse the above string and ignore errors?

**Comment by jaylinski** [comment]
I think there isn't. But you could wrap the `compile`-function in a try-catch block:

```
var template = '{{ test }} {{ test2';
var compile = Handlebars.compile(template);

try {
	console.log(compile({test: 'Max'}));
} catch (e) {
	console.log(template);
}
```

**Reply by stalingino** [reply]
This wouldn't solve the problem. I would need a way to parse whatever is valid & ignore remaining
